code/js/inject/devchat_inject.js

```diff
@@ -0,0 +1,52 @@
+(function() {
+  "use strict";
+
+  var sk_log = require("../modules/SKLog.js");
+
+
+  var audio = document.getElementsByTagName("audio")[0];
+
+  if(audio === undefined)
```

Comment:
To keep with the style used elsewhere (eg: https://github.com/berrberr/streamkeys/blob/master/code/js/modules/BaseController.js#L199), please write as:

```
if(audio === undefined) return null;
```